components/gfx/Cargo.toml

```diff
@@ -28,6 +28,7 @@ msg = {path = "../msg"}
 net_traits = {path = "../net_traits"}
 ordered-float = "0.4"
 range = {path = "../range"}
+script = {path = "../script"}
```

Comment:
This change is not necessary right now - there are no changes in this PR that add a `use script::` in gfx. What am I missing?